app/frontend/src/components/Welcome.vue

```diff
@@ -0,0 +1,98 @@
+<template>
```

Comment:
Think this page is called "About" in the latest revisions
https://www.figma.com/proto/PqqxnXfahmDGJYC74fj1bQ/onboarding_v5?node-id=30%3A60&scaling=min-zoom
Maybe check with Mina and see if that is looking final then rename.